_inc/lib/core-api/wpcom-endpoints/subscribers.php

```diff
@@ -40,8 +40,12 @@ public function readable_permission_check() {
 	 * @return array data object containing subscriber count
 	 */
 	public function get_subscriber_count( $request ) {
-		$subscriptions = new Jetpack_Subscriptions_Widget();
-		$subscriber_info = $subscriptions->fetch_subscriber_count();
+		// Get the most up to date subscriber count when request is not a test
+		if ( ! defined( 'TESTING_IN_JETPACK' ) ) {
```

Comment:
does Jetpack side knows when the site gets a new subscriber? could we delete the transient at that point?

Reply:
@lezama I don't believe it does. There is an `increment_subscriber_count` function, https://github.com/Automattic/jetpack/blob/d7a1fadb4163d62d388c1b636c05f261fbab31fd/modules/subscriptions.php#L984, but I searched the Jetpack repo and don't see it invoked anywhere. 

I believe the way it works is that subscribers confirm their subscription via the email they receive from wpcom, and the Jetpack site simply drops its transient and queries wpcom if the transient is more than an hour old to get the latest confirmed subscriber count.